exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -202,8 +202,7 @@ def _create_database_info(self, db_ip_address: str, reused: bool) -> DatabaseInf
                 network_info=self.network_info,
                 volume_name=self._get_db_volume_name(),
             )
-            assert self.ssh_key_file
-            ssh_info = SshInfo(self.ssh_user, str(self.ssh_key_file))
+            ssh_info = SshInfo(self.ssh_user, str(self.ssh_key_file or ""))
```

Comment:
Here is actually the question, how can this be empty and if it is empty what does it mean, should maybe in that case the whole SshInfo not exist

Reply:
Imho this whole Ssh key file handling should be refactored. 
```
self.ssh_key_file 
``` is overwritten in several places here and it's very confusing. However, I would not like to do the refactoring in this PR.